packages/perseus/src/perseus-types.ts

```diff
@@ -483,6 +483,9 @@ export type PerseusGradedGroupSetWidgetOptions = {
     gradedGroups: ReadonlyArray<PerseusGradedGroupWidgetOptions>;
 };
 
+// 2D range: xMin, xMax, yMin, yMax
+export type GraphRange = [[number, number], [number, number]];
```

Comment:
I recently learned that tuple members can have names in TypeScript! So you can do this:

```ts
export type GraphRange = [
  x: [min: number, max: number],
  y: [min: number, max: number],
];
```

This seems a bit more self-documenting.